sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py

```diff
@@ -0,0 +1,192 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import time
+import argparse
+import asyncio
+import os
+import logging
+from collections import defaultdict
+
+from azure.identity.aio import ClientSecretCredential
+from azure.eventhub.aio import EventHubConsumerClient
+from azure.eventhub import EventHubSharedKeyCredential
+from azure.eventhub.extensions.checkpointstoreblobaio import BlobCheckpointStore
+from azure.eventhub import TransportType
+
+from logger import get_logger
+from process_monitor import ProcessMonitor
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--link_credit", default=3000, type=int)
+parser.add_argument("--output_interval", type=float, default=1000)
+parser.add_argument("--duration", help="Duration in seconds of the test", type=int, default=30)
+parser.add_argument("--consumer_group", help="Consumer group name", default="$default")
+parser.add_argument("--auth_timeout", help="Authorization Timeout", type=float, default=60)
+parser.add_argument("--offset", help="Starting offset", default="-1")
```

Comment:
Change to event_position? Not necessary to be an offset

Reply:
added new args, now becoming
- starting_offset
- starting_sequence_number
- starting_datetime (str should be format of YYYY-mm-dd HH:mm:ss)